fixtures/refund.go

```diff
@@ -2,6 +2,10 @@ package fixtures
 
 import "github.com/companieshouse/payments.api.ch.gov.uk/models"
 
+var LateFilingPenalty = "LATE FILING PENALTY"
```

Comment:
Original text was mixed case "Late Filing Penalty".
Is this change correct?

Reply:
It's used as test data so not breaking change but I'll change it back anyway as it makes more sense